ts/components/portfolio/CreditCardComponent.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react"
+
+import {
+  Body,
+  Card,
+  Icon,
+  Text,
+} from "native-base";
+import { Image } from "react-native"
+import { Col, Grid, Row } from "react-native-easy-grid"
+import { NavigationScreenProp, NavigationState } from "react-navigation"
+
+import I18n from "../../i18n"
+import ROUTES from "../../navigation/routes"
+
+import { CreditCard } from "../../types/portfolio/CreditCard"
+import { CreditCardType } from "../../types/portfolio/CreditCardType"
+import { CreditCardStyle } from '../styles'
+
+type Props = {
+  item: CreditCard,
+  navigation: NavigationScreenProp<NavigationState>
+}
+
+// Images
+const imgMastercard = require("../../../img/portfolio/issuers/mastercard.png")
+const imgVisa = require("../../../img/portfolio/cards-icons/visa.png")
+const imgAmex = require("../../../img/portfolio/cards-icons/amex.png")
+const imgDiners = require("../../../img/portfolio/cards-icons/amex.png")
+const imgMaestro = require("../../../img/portfolio/cards-icons/amex.png")
+const imgElectron = require("../../../img/portfolio/cards-icons/visa-electron.png")
+
+/**
+ * Credit card component
+ */
+export default class CreditCardComponent extends React.Component<Props> {
+
+  public render(): React.ReactNode {
+    const { item } = this.props;
+    const { navigate } = this.props.navigation;
+    return (
+      <Card
+      style={ CreditCardStyle.cardStyle } >
+      <Body>
+        <Grid>
+          <Row size={2} style = {CreditCardStyle.rowStyle}>
+            <Col size={7}>
+              <Text style = { [CreditCardStyle.textStyle, CreditCardStyle.largeTextStyle] } >
+                {("\u25cf".repeat(4) + " ").repeat(3) + item.number.slice(-4)}
+              </Text>
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={false}
+                style={CreditCardStyle.iconStyle}
+                name="star-border"
+              />
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={true}
+                style={CreditCardStyle.iconStyle}
+                name="more-vert"
+              />
+            </Col>
+          </Row>
+          <Row size={1} style = {CreditCardStyle.rowStyle}>
+            <Col>
+              <Text style = { [CreditCardStyle.textStyle, CreditCardStyle.smallTextStyle] } >
+                {I18n.t('creditCardComponent.validUntil') + item.expires}
+              </Text>
+            </Col>
+          </Row>
+          <Row size={2} style = {[CreditCardStyle.rowStyle, CreditCardStyle.whiteBarStyle]}>
+            <Col size={7}>
+              <Text style = { CreditCardStyle.textStyle }>
```

Comment:
perhaps we can use a `text-transform: uppercase` in the style instead of the `.toUpperCase()`?

Reply:
Unfortunately `text-transform` is not supported yet (https://github.com/facebook/react-native/issues/2088) (actually, there appears to be some kind of support for iOS, but not android)